pkg/diff/diff.go

```diff
@@ -47,6 +47,8 @@ type EntityChanges struct {
 
 var errEnqueueFailed = errors.New("failed to queue event")
 
+const KindLicense = "license"
+
```

Comment:
I might be missing something but this looks unused.

```suggestion
```

Reply:
removed.